### PR TITLE
Add news for playtest 20170923.

### PR DESCRIPTION
**Pull request by pchote**
screenshot-2017-9-23 openra
screen shot 2017-09-23 at 16 03 49

### CODE REVIEW

**Comment by reaperrr** [comment]
Looks good to me 👍

**Comment by pchote** [comment]
* Updated github ID for the new tag
* Updated changelog commit
* Changed "today pushing" to "today releasing" to avoid repeating the same word twice.
* Moved the Mod SDK sentence to its own paragraph
* Added a final sentence linking to the downloads page.

This should now be ready to merge.

File: content/news/playtest-20170923.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+The [Mod SDK](https://github.com/OpenRA/OpenRAModSDK/releases/tag/20170923) has also been updated to use the new engine version, and to create "portable" windows packages that do not require installation.
+
+<div style="text-align:center" markdown="1">
+![New civilian tech buildings](/images/news/20170923-assaultmove.gif)
+
+Hold <span style="color: #FFD700;">Ctrl</span> when issuing an Attack Move order to activate Assault Move.
+</div>
+
+This playtest will be your last chance to report any bugs or balance issues, so we encourage everybody to play a few games, and let us know of any issues via [our forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), our [GitHub issue tracker](https://github.com/OpenRA/OpenRA/issues), or in the comments below.  As usual, you can find playtest-20170902 on our [download page](http://www.openra.net/download/).
```
Comment (reaperrr) [inline content/news/playtest-20170923.md]: Last sentence still says playtest-20170902.

**Comment by reaperrr** [comment]
Other than that looks good to me.

**Comment by pchote** [comment]
Oops.  Fixed.

**Pull request merged**